astro.config.mjs

```diff
@@ -7,4 +7,10 @@ export default defineConfig({
   base: "",
   integrations: [mdx(), solidJs()],
   output: "static",
+  site: "https://maplibre.org/",
```

Comment:
noting that this is also needed in a few places by the RSS feed, could look at another approach if you'd prefer.